.storybook/manager-head.html

```diff
@@ -1,44 +1 @@
 <!-- reference for customizing https://github.com/storybookjs/storybook/issues/11222#issuecomment-646411961 -->
-<link rel="stylesheet" href="https://nav-prod.home.static.arcpublishing.com/css/arcnav.css" />
```

Comment:
There are no current plans to not render this in ALC going forward - while that could change, we should keep this until we decided either way

Reply:
ok will revert

Reply:
updated